### ISSUE TITLE
Create an accessibility landing / routing page at jupyter.org/accessibility

### ISSUE THREAD

**Issue by choldgraf** [jupyter/accessibility#154]
Right now, Jupyter doesn't have a single obvious landing page to help people understand where they can learn more about accessibility. That said, there are a lot of different user interfaces in Jupyter and its subprojects, and also different definitions of "accessibility". People have a hard time navigating all of these spaces, and are often unsure what is authoritative, what is up to date, etc.

I suggest that we create a single "landing page" that helps point people in the right direction, at an authoritative and easy to remember URL.

### To do / questions

- [ ] Is this overall idea reasonable? Any objections?
- [ ] Any changes you'd make to the structure below?

### Suggestion: jupyter.org/accessiblity

This would be a landing page to frame the general goals for accessibility across Jupyter, and provide links to places where readers can learn more. It probably wouldn't be an evergreen page with constantly updated content, but would instead link to other evergreen places that we expect to be more up to date. For example:

```
[Statement of accessibility goals across Jupyter].
[Honest take on where we are right now (e.g., we've made progress, and still have plenty of improvements to make, something like that)]

## Accessibility subproject provides cross-project guidance and strategy

[A few bullets about the purpose of this group and its role in jupyter, a link to learn more].

[link to the accessibility subproject docs (is that still the readthedocs site?)]

## Where to engage around accessibility

[Links to the meetings or spaces where accessibility is discussed]
[E.g., a github issue tag, zulip channel, calendar meeting, etc]

## Project-specific accessibility documentation

[A statement about how subprojects manage their own accessibility work]
[A list of links to each one that Chris knows of, e.g.:

- jupyter book's accessibility page (https://mystmd.org/guide/accessibility-and-performance)
- similar pages for jupyter hub or lab
```

### What this would *not* be

- A place with full information about accessibility, tools, practices, etc. IMO that's the accessibility subproject docs.
- A place with up to date information about the state of accessibility of each subproject (instead, it'd link to _their_ accessibility statements)

**Comment by sean-morris** [comment]
I support all this effort. Currently, I am reluctant to send potential users to the Accessibility page linked from jupyter.org mainly because the overriding message is we are not accessible. I support being honest about where are but it would be great to also detail what is accessible and what kind of plan we might have in place to meet a11y guidelines.

**Comment by penn5** [comment]
For me, having a signpost saying "this software is not accessible" somewhere would actually be very helpful. For example, it would support colleagues, teachers and officials with understanding that alternative formats must be provided when distributing Jupyter files. It would also save people time trying to figure out whether Jupyter is accessible, and could bring new contributors in some cases. Having a list of workarounds there would be nice if any are known beyond the hacky one I recently posted in discussions.

**Comment by choldgraf** [comment]
Here's a recent example of a similar kind of page we put together for Jupyter Book / MyST:

https://jupyterbook.org/accessibility/

Personally, I think we should define our accessibility goals, note any obvious places where we aren't meeting them, and note how people can help and follow along. We should also break this down by sub-project since there is a lot of different tech in Jupyter at varying levels of accessibility. That feels more useful than a single blanket statement for the whole project.
